charts/newrelic-k8s-metrics-adapter/templates/deployment.yaml

```diff
@@ -18,22 +18,22 @@ spec:
         {{- toYaml .Values.podAnnotations | nindent 8 }}
         {{- end }}
       labels:
-        {{- include "newrelic-k8s-metrics-adapter.labels" . | nindent 8 }}
+        {{- include "common.labels" . | nindent 8 }}
     spec:
-      serviceAccountName: {{ template "newrelic-k8s-metrics-adapter.serviceAccountName" . }}
+      serviceAccountName: {{ include "common.serviceAccount.name" . }}
       {{- if .Values.podSecurityContext.enabled }}
       securityContext:
         runAsUser: {{ .Values.podSecurityContext.runAsUser }}
         runAsGroup: {{ .Values.podSecurityContext.runAsGroup }}
         fsGroup: {{ .Values.podSecurityContext.fsGroup }}
```

Comment:
[This PR is still pending](https://github.com/newrelic/helm-charts/pull/725).

Reply:
Fixed in last version